src/interfaces/IGovernorAlpha.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
```

Comment:
*(arbitray place)*

- This interface is the same as gitcoin sans the `gtc` method
- I also verified this matches the Pooltogether governor alpha